local/config/vufind/searchspecs.yaml

```diff
@@ -356,6 +356,8 @@ AllFields:
     - description
     - isbn
     - issn
+    - key_word_chains
+    - key_word_chain_bag
```

Comment:
Seems necessary from my point of yiev to allow jumping from KWC search to ordinary search